docs/appendix/yaml.rst

```diff
@@ -286,6 +286,13 @@ observation label is first and the model label is second
 
 **data_proc:** This section stores all of the data processing information.
 
+   * **rem_obs_by_nan_pct:** Specify as dictionary with keys 'group_var', 
+     'pct_cutoff' and 'times'. If specified, removes all instances of 
+	 'group_var' where there are > 'pct_cutoff' % NaN values. For example, 
+	 with airnow sites, setting 'group_var' to 'siteid' will remove all 
+	 sites with > pct_cutoff NaN values. Setting 'times' to 'hourly' will 
+	 only look at values at the beginning of each hour. Set 'times' to ''
+	 if all times should be used.
```

Comment:
align this block with L edge of previous line to (hopefully) please the docs build CI